src/test/java/tests/LoginTests.java

```diff
@@ -21,6 +21,16 @@ public void tc01_LoginWithValidData() {
         loginPage.login(Constants.USER, Constants.PASSWORD);
 
         LogHelper.info("Check welcome user message");
-        Assert.assertEquals(loginPage.getWelcomeUserMessage(), "Welcome " + Constants.USER, "Welcome user message doesn't display correctly.");
+        Assert.assertEquals(loginPage.getWelcomeUserMessage(), "Welcome " + Constants.USER, "Welcome user message doesn't display correctly");
+    }
+
+    @Test(description = "User can't login with blank 'Username' textbox")
+    public void tc02_LoginWithBlankUsernameTextbox() {
+        LogHelper.info("Don't enter user name. enter valid information into 'Password' textbox");
```

Comment:
It should be an uppercase letter after dot

Reply:
I've already fixed it.